src/api/todos.ts

```diff
@@ -2,7 +2,20 @@ import { Todo } from '../types/Todo';
 import { client } from '../utils/fetchClient';
 
 export const getTodos = (userId: number) => {
-  return client.get<Todo[]>(`/todos?userId=${userId}`);
+  return client.get<Todo[]>(`/todos?userId=${userId}`) || null;
```

Comment:
why `null` is needed here? it could cause a potential error if you try to call `.then` method for `getTodos` function